src/client/sky.h

```diff
@@ -33,6 +33,7 @@ class Sky : public scene::ISceneNode
 	virtual void OnRegisterSceneNode();
 
 	//! renders the node.
+	virtual void renderTextures(video::IVideoDriver *driver);
```

Comment:
please move this to the `private` section, as this isn't part of the public interface and just used in the implementation of `render`